internal/pkg/cli/env_init.go

```diff
@@ -141,12 +141,12 @@ type initEnvVars struct {
 	isProduction  bool   // True means retain resources even after deletion.
 	defaultConfig bool   // True means using default environment configuration.
 
-	importVPC importVPCVars // Existing VPC resources to use instead of creating new ones.
-	adjustVPC adjustVPCVars // Configure parameters for VPC resources generated while initializing an environment.
-	telemetry telemetryVars // Configure observability and monitoring settings.
-
-	tempCreds tempCredsVars // Temporary credentials to initialize the environment. Mutually exclusive with the profile.
-	region    string        // The region to create the environment in.
+	importVPC   importVPCVars // Existing VPC resources to use instead of creating new ones.
+	adjustVPC   adjustVPCVars // Configure parameters for VPC resources generated while initializing an environment.
+	telemetry   telemetryVars // Configure observability and monitoring settings.
+	importCerts []string      // Addtional existing ACM certificates to use.
+	tempCreds   tempCredsVars // Temporary credentials to initialize the environment. Mutually exclusive with the profile.
```

Comment:
nit: would you mind adding a newline after `importCerts` to separate the `tempCreds` group from the `importVPC` group
```suggestion

	tempCreds   tempCredsVars // Temporary credentials to initialize the environment. Mutually exclusive with the profile.
```